.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-20.04
+    env:
+      MIKE_VERSION: v3
```

Comment:
For v2 branch those would need to be changed to:
`MIKE_VERSION: v2`
`MIKE_ALIAS: stable`